04.class/lib/view.js

```diff
@@ -0,0 +1,29 @@
+import enquirer from "enquirer";
+
+export class View {
+  #MODE_TABLE = { r: "see", d: "delete" };
```

Comment:
read/writeをr/wと略すことはありますが、dがdeleteっていうのはあまり見かけないかも。
オレオレで略さずに素直に全部書いてあげた方がいいんじゃないかなー。
```suggestion
  #MODE_TABLE = { read: "see", delete: "delete" };
```

https://blog.jnito.com/entry/2020/10/20/092724

Reply:
コマンドラインのオプション`-r`と`-d`に対応させたかったのですが、わかりにくかったですね💦
だとしてもオプションの情報が`memo_app.js`と`view.js`の両方に散らばることになるので良くないですね💦